### PR TITLE
Add WithFeatureFlag for landscape chat

### PR DESCRIPTION
**Pull request by jordanwade**
This sets up the useFeatureFlag hook so we can use it in class components and adds a feature flag check to the landscape chat feature.

image

### CODE REVIEW

File: src/media-player/controls/LiveStreamControls.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ import {
   MUTE,
   UNMUTE,
 } from '../mutations';
+import { WithFeatureFlag } from '../../hooks';
```
Comment (JSGraef) [inline src/media-player/controls/LiveStreamControls.js:38]: I don't think you need to have a relative import here. It can just be `'hooks'`.

**Review by JSGraef — APPROVED** [review]
👍

File: src/media-player/controls/LiveStreamControls.js (hunk 1 above)
Comment (rkdavidson) [reply src/media-player/controls/LiveStreamControls.js:38]: I can't remember for sure, but I _think_ that import style/setup is only for the web unfortunately @JSGraef .
IIRC It's something with the `create-react-app` config or a customization of it.

I'd love to bring whatever mechanism lets us do that though, so the app can get rid itself of the relative import path plague.

**Review by rkdavidson — APPROVED** [review]

File: src/media-player/controls/LiveStreamControls.js (hunk 1 above)
Comment (JSGraef) [reply src/media-player/controls/LiveStreamControls.js:38]: Pretty sure the mechanism is already there. I see 7 other instances of just referencing `'hooks'`. Small thing anyway that doesn't matter 🤷‍♂️

Comment (rkdavidson) [reply src/media-player/controls/LiveStreamControls.js:38]: Oh, nice! Yeah, good item for F&I to do a targeted warnings and imports cleanup.

**Pull request merged**